content/references/proptypes.md

```diff
@@ -78,6 +80,39 @@ export const defaultConfig = {
 
 Merchant edits to color fields are stored in RGBA format.
 
+### date
+
+![Date Picker](proptype-date.png)
+
+```javascript
+date: {
+  label: 'Date',
+  type: ElementPropTypes.date
+}
+
+export const defaultConfig = {
+  date: new Date(2020, 2, 16)
```

Comment:
```suggestion
  date: new Date('March 16, 2020')
```

I know what you had before is also acceptable, but a zero index month is an easy trip up to make.  I know I've done it more than once.

Also, I wonder if we should maybe add a note about suggesting using `new Date()` as the default so that it always starts at today when adding the block to the page (I assume it would work that way and not just save the date at time of release)?

Reply:
This is a good idea.